content/general/concepts/tech-stacks/tech-stacks.md

```diff
@@ -0,0 +1,82 @@
+
+- Javascript: JavaScript is the programming language of the Web. It is used to make web pages interactive and it allows the implementation of dynamic web pages. Some of the popular javascript frameworks include React, Vue, Angular and svelte.
+
+### Backend Tech Stack
+The back end also known as the server-side refers to parts of a computer application or a program's code that allow it to operate and that cannot be accessed by a user. The back end of a computer system is where the majority of data and operating syntax are kept and accessible.
+
+The backend tech stack comprises the following elements:
+  Programming Languages: A programming language is a computer language programmers use to develop software programs or other sets of instructions for computers to execute. Examples include Javascript, Python, Java, etc.
+
+- Frameworks: A framework is a tool that provides ready-made components or solutions that are customized in order to speed up development. Popular frameworks used in development include Laravel, Ruby on Rails, Express, and Django.
+
+- Databases: A database is a systematic collection of structured information or data. The most common databases include MongoDB, Postgresql, MySQL, and SQL.
+
+- Servers: A server is a computer program or device that provides a service to another computer program and its user, also known as the client. It responds and accepts to requests made over a network. Commonly used servers are AWS, Google Cloud, Apache, Nginx, Microsoft's Internet Information Services (IIS), etc.
+
+
+### LAMP Stack
+The LAMP stack is a popular open-source tech stack that is used to create web apps. It is one of the oldest tech stacks. The LAMP stack provides a collection of software for delivering high-performance web applications. This stack comprises the following technologies
+
+- L — Linux: An open-source operating system based on the Linux kernel
+- A — Apache: A free and open-source cross-platform web server.
+- M — MySQL: A relational database management system for storing and manipulating data.
+- P — PHP: A server-side scripting programming language used for creating dynamic web applications.
+
+### MERN Stack
+MERN is used to describe a specific set of JavaScript-based technologies that are used in the web application development process. It is made of technologies such as MongoDB, Express, React, and Node. The MERN stack is one of the most popular technology stacks.
+
+- M – MongoDB: MongoDB is a document database used to build highly available and scalable internet applications.
+- E – Express(.js): A backend framework built on Nodejs and it provides features for developing mobile and web applications.
+- R — React(.js):  A frontend javascript library developed by Meta for building user interface.
+- N — Node(.js): A javaScript runtime environment that runs on a JavaScript Engine.
```

Comment:
```suggestion
- MongoDB is a document database used to build highly available and scalable internet applications.
- Express(.js) is a back-end framework built on Node.js and it provides features for developing mobile and web applications.
- [React(.js)](https://www.codecademy.com/resources/docs/react) is a front-end library developed by Meta for building user interface.
- Node(.js) is a runtime environment that runs JavaScript on the server-side.
```

We do have existing entries for Express.js and Node.js but I'm not going to suggest linking them at this time because they are in the process of the being moved and having their URLs changed.